src/Schema/Events/ImpressionTracking.ts

```diff
@@ -123,7 +123,9 @@ export interface ErrorMessageViewed {
  *    context_page_owner_type: "orders-shipping",
  *    context_page_owner_id: "57e60c68-a198-431e-8a02-6ecb01e3a99b",
  *    user_id: "61bcda16515b038ce5000104"
- *    flow: "user adding shipping address"
+ *    flow: "user adding shipping address",
+ *    subject: "Check your delivery address",
+      option: "review and confirm",
```

Comment:
Thank you! Here is the [relevant thread](https://artsy.slack.com/archives/C02JHHHKP5K/p1689837587009359?thread_ts=1689786251.379909&cid=C02JHHHKP5K) to finalize the new properties.